src/convertToObject.js

```diff
@@ -12,7 +12,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newArr = sourceString
+  .split(';')
+  .map(val => val.trim().split(':'))
+  .filter(val => val.length > 1);
```

Comment:
code it not aligned. 
```suggestion
  const newArr = sourceString
    .split(';')
    .map(val => val.trim().split(':'))
    .filter(val => val.length > 1);
```